### ISSUE TITLE
Blackhole crash

### ISSUE THREAD

**Issue by SnakeZzzzz** [BespokeSynth/BespokeSynth#1170]
Opening blackhole plugin it automatically closes bespokesynth

**Comment by NoxyNixie** [comment]
Make sure you are opening the 64bit version of blackhole.

Do you have a crashlog (they can be found in `Documents/BespokeSynth`)?

**Comment by SnakeZzzzz** [comment]
Yes, I am opening the AU. Even when opening Endlesss studio plugin (in Bespoke everything ok), and whitin Endlesss trying Eventide Blackhole it will close abruptly. No crash report is generated :(

In Endlesss standalone I can open Blackhole without issues

**Comment by NoxyNixie** [comment]
There should always be a crashlog. They are named `crash_<datetime>.txt` and should be in your `Documents/BespokeSynth` folder, one folder down from the default location with savestates.

(Unless something goes very horribly wrong in which case this is likely a system issue not a Bespoke issue)

**Comment by SnakeZzzzz** [comment]
Thank you for your guidance. Yes, it seems to be dramatic and strange. I cannot even open the plugin (the instant I select it from the plugins list, bespoke is closed). So no crash report is generated. It is the 64bit version. Reinstalled again and same result.

**Comment by NoxyNixie** [comment]
Maybe this is an architecture mismatch? I know new macs use this new M1 architecture that isn't exactly compatible with the old. Bespoke should be compatible (at least the Nightly builds are) but maybe your blackhole plugin isn't?

**Comment by SnakeZzzzz** [comment]
I have this info from another crash (same computer, other plugin), regarding OS, version... 
Still, Blackhole plugin closes Bespoke abruptly, no crash file is generated for it. Only for you to know system info.

OS: Mac OSX 13.1
CPU vendor: GenuineIntel
CPU model: Intel(R) Core(TM) i5-8257U CPU @ 1.40GHz
CPU speed: 1400 MHz
num cores: 8
num CPUs: 4
RAM: 8192 MB
computer: Pablos-MacBook-Pro
language: es
region: ES
display language: es-ES
description: MacBookPro15,4
manufacturer: Apple
build: bespoke NIGHTLY 1.1.0 (Feb 17 2023 05:03:44))
